packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -95,3 +89,41 @@ export function RepeatableCheckbox(props: CommonCheckboxProps) {
     </Box>
   );
 }
+
+export function Variant() {
+  return (
+    <Box overflow="hidden" flexDirection="column" border borderRadius={6}>
+      <Box
+        border={{ bottom: true }}
+        padding={12}
+        flexDirection="column"
+        gap={8}
+      >
+        <Text variant="h4" color="grey12">
+          Variants
+        </Text>
+        <Text color="grey11">
+          Add variants like "Primary" and "Secondary" to allow editors to choose
```

Comment:
⚠️ For me I don't think it's the correct color corresponding to the Figma.

Note: At the same time, I couldn't find the correct grey value from Figma, seems weird and not coming from the DS. Maybe we could just took the darker version of this as visible in the design.

Reply:
yeah, the color doesn't exist in the new design system, so i picked the closest one that looked right

>Maybe we could just took the darker version of this as visible in the design

not sure i understood this message, could you clarify?

Reply:
From the design, I feel the color should be darker. Maybe we could use one a darker to be closer to the design?
Since this one is the same as the others texts it's too close and don't match design.

Reply:
i see what you mean now, i've used the darker color now, let mem know what you think!